analyzers/tests/SonarAnalyzer.UnitTest/ResourceTests/FilesToAnalyze/FilesToAnalyze.txt

```diff
@@ -0,0 +1,5 @@
+﻿C:\Projects/DummyProj/wEB.config
```

Comment:
maybe also add a test case with file with some numbers or non-path text? (like `foo`, `123`)

or add it here directly if the class supports random stuff inside